src/fuzzers/bash/oss-fuzz-build.sh

```diff
@@ -38,16 +38,25 @@ mkdir -p "$outputDir"
 # Switch to the project directory (by referencing from the script directory)
 cd -- "$(dirname -- "${BASH_SOURCE[0]}")" && cd ../../..
 
-# Build the project
-mkdir -p build && cd $_
-cmake \
-  -DBUILD_SHARED_LIBS=OFF \
-  -DBUILD_TESTING=OFF \
-  -DBUILD_EXAMPLES=OFF \
-  -DBUILD_FUZZERS=ON \
-  ..
-cmake --build . -j
-
-# Copy executables & resources
-find src/fuzzers/cpp -maxdepth 1 -executable -type f -exec cp -v {} "$outputDir/" \;
-cp -v ../src/fuzzers/resources/* "$outputDir/"
+# Build a fuzzer with each encoding
+declare -a encodings=("utf-8" "wchar_t")
+
+for encoding in "${encodings[@]}"
+do
+  echo heeeeere
```

Comment:
debug message?

Reply:
Yes, thanks for catching! Fixed in https://github.com/apache/logging-log4cxx/pull/465/commits/c8cb362b03f1e3bf6b9b0af071cd8c3ba43efd79